browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -114,6 +114,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		this._controlHandlers['spinnerimg'] = this._spinnerImgControl;
 		this._controlHandlers['image'] = this._imageHandler;
 		this._controlHandlers['scrollwindow'] = this._scrollWindowControl;
+		this._controlHandlers['scrollbarbox'] = this._ignoreHandler;
```

Comment:
I see no effect of this, what does it fix?

Reply:
Accidentally left in, ignore.